### ISSUE TITLE
User account vanished

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#4293]
**[Submitted to the original trac issue database at 2.49am, Tuesday, 13th March 2012]**

Same user as in ticket #4290

http://www.openstreetmap.org/user/here

Was able to log in and de-anonymize the account some hours ago and was logged in for a while. Now the account has suddenly (partially) vanished. Gets "username/email address doesn't exist" errors when trying to log in or reset password but also can't create a new account with the same name/email because it already exists.

I remember hitting the user page yesterday before the de-anonymizing. It showed no edits but was browsable. Today it gives a "user does not exist" error message.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 7.12am, Tuesday, 13th March 2012]**

It didn't "suddenly vanish", it was suspended by the antispam code when he changed his description and he was told to contact the webmaster for assistance.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 7.12am, Tuesday, 13th March 2012]**

Obviously I have unlocked the account now.

**Comment by tomhughes** [comment]
**Author: here**
**[Added to the original trac issue at 4.44am, Friday, 16th March 2012]**

I did change my description to include a link to my home web presence and a wikipedia presence.  I did not ever see a reference to contact the webmaster for assistance, nor any indication of suspension.  I've just searched through my email again, spam, trash, etc, and found no reference to any such recommendation.  Further, upon attempting login, there was again no indication of suspension or suggestion to contact a "webmaster".

Reopening only to help diagnose a confusing situation which confused both a new and experienced user.  What caused the suspension and why was that not caught in a more graceful manner?  What did we both miss in identifying the problem?  Where was the notification?  How can this be made more user friendly?

Looking forward to contributing, thanks for your help.
